main.go

```diff
@@ -1,7 +1,14 @@
 package main
 
-import "fmt"
+import (
+	"fmt"
+
+	_ "github.com/iterableio/api/config"
```

Comment:
Go will automatically alphabetize. I kind of like the way it is now. Better to group by source of lib rather than whether it's used or not.